pkg/apis/serving/v1beta1/route_defaults.go

```diff
@@ -20,5 +20,28 @@ import (
 	"context"
 )
 
+// SetDefaults implements apis.Defaultable
 func (r *Route) SetDefaults(ctx context.Context) {
+	r.Spec.SetDefaults(withinSpec(ctx))
+}
+
+// SetDefaults implements apis.Defaultable
+func (rs *RouteSpec) SetDefaults(ctx context.Context) {
+	if len(rs.Traffic) == 0 && hasDefaultConfigurationName(ctx) {
+		rs.Traffic = []TrafficTarget{{
+			Percent: 100,
```

Comment:
I like setting LatestRevision here for clarity rather than defaulting our default further.

Reply:
Done